src/client/multiStepInput.ts

```diff
@@ -0,0 +1,273 @@
+import {
+  QuickPickItem,
+  window,
+  Disposable,
+  QuickInputButton,
+  QuickInput,
+  QuickInputButtons
+} from "vscode";
+
+export async function multiStepInput() {
```

Comment:
Input should abort on cancel. Right now it cannot exit set-credentials workflow.

Reply:
Good call, I forgot to add that. Let me update this so ESC will cancel the operation.

Reply:
I have resolved this issue. ESC will now cancel the input process and exit the set-credentials workflow.